src/compat_tests.rs

```diff
@@ -24,8 +24,9 @@ use signature::{self, Verifying};
 #[cfg(feature = "signature-pgp")]
 mod pgp {
     use super::*;
+    use futures::io::AsyncWriteExt;
     use signature::pgp::{Signer, Verifier};
-    use tokio::io::AsyncWriteExt;
+    use tokio_util::compat::TokioAsyncReadCompatExt;
 
     #[tokio::test]
     #[serial_test::serial]
```

Comment:
SHouldn't this be bounded by `async-tokio` as well, probably within another nested module level?

Reply:
No, I don't think this is necessary. `async-tokio` isn't a feature flag any more and `tokio` is always used in dev-dependencies.